requirements/jax/env_jax.yml

```diff
@@ -4,5 +4,11 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - numpy==1.19.*
+  - numpy==1.21.*
+  - rdkit
```

Comment:
Should rdkit be a dependency here?

Reply:
In my tests, I'm using `delaney` & `bace` datasets from `molnet` which uses circular fingerprints.